guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -33,24 +35,57 @@ the upgrade to proceed.
 +
 For information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
 
-. Optional: If you made manual edits to DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, back up the configuration files because the installer only supports one domain or subnet, and therefore restoring changes from these backups might be required.
-
-. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
+ifdef::katello[]
++
+. Regenerate certificates.  On the main Foreman/Katello server:
```

Comment:
Please convert to one sentence per line & use asciidoc attributes instead of `Foreman`: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L80

Reply:
@maximiliankolb this guide is new and was brought upstream quite fast. I think there are probably problems with one sentence per line rule violations throughout. If it is the case, I can spend a while later today fixing this rather than blocking this PR (It's kind of my fault for not doing this upfront but this is the last guide to come upstream).

Would you be OK with that?